### PR TITLE
Release 1.0.21

### PR DESCRIPTION
**Pull request by brandonpollack23**
Push a release so that pulumi/pulumi can move forward

### CODE REVIEW

File: pkg/convert/pulumiverse.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ var pulumiSupportedProviders = []string{
 	"kafka",
 	"keycloak",
 	"kong",
+	"kubernetes",
```
Comment (tgummerer) [inline pkg/convert/pulumiverse.go:59]: This change seems unintentional here?  It should probably be in a different PR than the release PR.

Comment (brandonpollack23) [reply pkg/convert/pulumiverse.go:59]: This PR is just on top of that one so I don't submit them in the wrong order, just filter by latest commit :)

**Review by tgummerer — APPROVED** [review]
